homeassistant/components/home_connect/quality_scale.yaml

```
@@ -0,0 +1,93 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling:
+    status: todo
+    comment: |
+      The integration polls the Home Connect API every minute. This frequency can be lowered
+      because we receive push updates from the API so we don't need to poll that often.
+  brands: done
+  common-modules:
+    status: todo
+    comment: |
+      Coordinator is not present in this integration but should be.
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  entity-event-setup:
+    status: exempt
+    comment: |
+      Entities of this integration does not explicitly subscribe to events.
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: todo
+  unique-config-entry:
+    status: exempt
+    comment: |
+      The Home Connect API does not allow to identify a user, so there is no
+      way to know if multiple config entries are using the same account.
+
+  # Silver
+  action-exceptions:
+    status: todo
+    comment: |
+      The integration does not raise exceptions when an action fails.
+  config-entry-unloading: done
+  docs-configuration-parameters: done
+  docs-installation-parameters: done
+  entity-unavailable:
+    status: todo
+    comment: |
+      Some entities are not marked as unavailable in cases it should be.
+  integration-owner: done
+  log-when-unavailable: todo
+  parallel-updates: done
```

looks like it's missing to me
```suggestion
  parallel-updates: todo
```